src/coverlet.core/Symbols/CecilSymbolHelper.cs

```diff
@@ -1274,6 +1274,61 @@ private bool SkipExpressionBreakpointsSequences(MethodDefinition methodDefinitio
             return false;
         }
 
+        public bool SkipInlineAssignedAutoProperty(bool skipAutoProps, MethodDefinition methodDefinition, Instruction instruction)
+        {
+            if (!skipAutoProps || !methodDefinition.IsConstructor) return false;
+
+            return SkipGeneratedBackingFieldAssignment(methodDefinition, instruction) ||
+                   SkipDefaultInitializationSystemObject(instruction);
+        }
+
+        private static bool SkipGeneratedBackingFieldAssignment(MethodDefinition methodDefinition, Instruction instruction)
```

Comment:
do you mind to move all these static to instance method? Now that is a "service" with public contract we can move to instances method.

Reply:
Sure but those two methods don't access instance data or call other instance methods. Thus I can make them ```private static```. I was told that this can have a performance benefit?

Reply:
Yep I think small improvement(no instance pointer to pass), I don't like too much `static` when I see it every time I need to check if there is some shared state somewhere 😄 , anyway it's not fundamental, we can go with it.